readme.rst

```diff
@@ -102,17 +102,53 @@ This will do several things:
 The ``run`` target is therefore very convenient when preparing to submit a
 change to the documentation.
 
-Submit your change
-~~~~~~~~~~~~~~~~~~
+Local Checks
+~~~~~~~~~~~~
+
+Before committing and pushing changes, it's a good practice to run various checks locally to catch issues early in the development process.
 
-Prior to submitting your change, it is recommended to do a fresh build in order
-to surface any errors that may cause build issues on the RTD side:
+Local build
+^^^^^^^^^^^
+
+Build the docs as HTML to surface any build errors that would occur in RTD:
```

Comment:
The important thing here is to do a clean/fresh build, not that you build HTML.
People have been building HTML all the time while developing, but they might have missed some errors (which won't show again because of incremental builds).

Reply:
Yep makes sense. Done.